lib/broadside/deploy.rb

```diff
@@ -4,9 +4,11 @@ class Deploy
 
     attr_reader :target
 
+    delegate :tag, to: :target
+
     def initialize(options = {})
       @target = Broadside.config.get_target_by_name!(options[:target])
-      @tag = options[:tag] || @target.tag
+      @target.tag = options[:tag] if options[:tag]
```

Comment:
whoops, i think i meant to link [this](https://github.com/lumoslabs/broadside/blob/981bf9be41c8339118610d301c7f9aec7c45dc4c/lib/broadside/deploy.rb#L9)

> only the Deploy object has the correct tag

right

> nothing appears to be using the Deploy object's tag, AFAICT.

wrong; [_everything_ uses the `Deploy` object `tag`](https://github.com/lumoslabs/broadside/blob/master/lib/broadside/ecs/ecs_deploy.rb#L205)